pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -181,6 +189,7 @@ func vmStatsCollectorCallback() []operatormetrics.CollectorResult {
 	}
 
 	var results []operatormetrics.CollectorResult
+	results = append(results, CollectVMCreatedTime(vms)...)
```

Comment:
totally a nit, but you added the metric as the last item of the list above, in the test file is also the last, but here the collect is the first. Can you move it to the bottom?

Reply:
so all here is from last to first, you want me to change the order for all?

Reply:
cc @machadovilaca

Reply:
yes, but maybe a different PR :raised_hands:

Reply:
sure so for this pr I fixed only the metric related.